package.json

```diff
@@ -205,8 +205,6 @@
         "esbuild-sass-plugin>esbuild": "^0.16.7",
         "react-electron-web-view>react": "^18.0.0",
         "react-electron-web-view>react-dom": "^18.0.0",
-        "react-loader>react": "^18.0.0",
-        "react-loader>react-dom": "^18.0.0",
```

Comment:
These two are not upgradable?

Reply:
`peerDependencyRules.allowedVersions` is a feature by pnpm to ignore dependencies which have incompatible peer requirements.

`react-loader` which is unmaintained only allowed react >=17, so we hacked it here. Since `react-loader` was replaced by a modern alternative in https://github.com/ferdium/ferdium-app/pull/1410, I was able to get rid of this now redundant override ;)